openedx/core/djangoapps/notifications/models.py

```diff
@@ -96,6 +96,8 @@ class Notification(TimeStampedModel):
     notification_type = models.CharField(max_length=64)
     content_context = models.JSONField(default=dict)
     content_url = models.URLField(null=True, blank=True)
+    web = models.BooleanField(default=True, null=False, blank=False)
+    email = models.BooleanField(default=True, null=False, blank=False)
```

Comment:
it makes sense to have `default=True` for the `web` column, but for `email` setting `default=True` would give the illusion that all previous notifications were emailed as well! 
I understand it would be a workaround for the time being, but the database would not reflect the `true` state of previous notifications! 
Is there a approach we where we can set this to `False` instead for existing notifications?